docs/source/guides/logical_types_and_semantic_tags.ipynb

```diff
@@ -0,0 +1,693 @@
+    "- Student Grade (A, B, C, D, F)\n",
+    "- Size (small, medium, large)\n",
+    "\n",
+    "##### PostalCode\n",
+    "\n",
+    "Represents Logical Types that contain a series of postal codes for representing a group of addresses.\n",
+    "\n",
+    "- **physical type**: `category`\n",
+    "- **standard tags**: `{'category'}`\n",
+    "- **koalas note**: Koalas does not support the `category` dtype, so for Koalas DataFrames and Series, the `string` dtype will be used.\n",
+    "\n",
+    "##### SubRegionCode\n",
+    "\n",
+    "Represents Logical Types that use the ISO-3166 standard sub-region code to represent a portion of a larger geographic region. ISO 3166-2 (sub-regions) codes are supported. These codes should be in the Alpha-2 format.\n",
+    "\n",
+    "- **physical type**: `category`\n",
+    "- **standard tags**: `{'category'}`\n",
+    "- **koalas note**: Koalas does not support the `category` dtype, so for Koalas DataFrames and Series, the `string` dtype will be used.\n",
+    "\n",
+    "For example: `'US-IL'` to represent Illinois in the United States or `'AU-TAS'` to represent Tasmania in Australia."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "obvious-mayor",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "categoricals_df = pd.DataFrame({\n",
+    "    'categorical': pd.Series(['a', 'b', 'a', 'a'], dtype='category'),\n",
+    "    'ordinal' : ['small', 'large', 'large', 'medium'],\n",
+    "    'country_code' : [\"AU\", \"US\", \"UA\", \"AU\"],\n",
+    "    'postal_code': [\"90210\", \"60035\", \"SW1A\", \"90210\" ],\n",
+    "    'sub_region_code' : [\"AU-NSW\", \"AU-TAS\", \"AU-QLD\", \"AU-QLD\"]\n",
+    "})\n",
+    "categoricals_df.ww.init(logical_types={'ordinal':ww.logical_types.Ordinal(order=['small', 'medium', 'large']),\n",
+    "                                       'country_code':'CountryCode', \n",
+    "                                       'postal_code':'PostalCode',\n",
+    "                                       'sub_region_code':'SubRegionCode'})\n",
+    "\n",
+    "categoricals_df.ww"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "elegant-saying",
+   "metadata": {},
+   "source": [
+    "#### Miscellaneous Logical Types with Specific Formats\n",
+    "\n",
+    "##### Boolean\n",
+    "\n",
+    "Represents Logical Types that contain binary values indicating true/false.\n",
+    "\n",
+    "- **physical type**: `bool`\n",
+    "\n",
+    "##### BooleanNullable\n",
+    "Represents Logical Types that contain binary values indicating true/false. May also contain null values.\n",
+    "- **physical type**: `boolean`\n",
+    "\n",
+    "##### Datetime\n",
+    "A Datetime is a representation of a date and/or time. Datetime variable types can be represented as strings, or integers. However, they should be in a intrepretable format or properly cast before using DFS. \n",
```

Comment:
`...before using DFS.` Should this be something else?

Reply:
whoopse, a holdover from the variable page. thanks for pointing out!